src/TodoModel.cpp

```diff
@@ -0,0 +1,48 @@
+#include "todo_lib/TodoModel.h"
+
+//Constructors
+TodoModel::TodoModel() {
```

Comment:
This shouldn't be empty, should assign default values for the properties. Especially not to forget all constructors must assign a unique id.